tests/identify_test.py

```diff
@@ -145,6 +145,47 @@ def test_tags_from_path_plist_text(tmpdir):
     }
 
 
+@pytest.mark.parametrize(
+    ('interpreter', 'expected'),
+    (
+        ('dson', {'salt', 'file', 'non-executable', 'text'}),
+        ('genshi', {'salt', 'file', 'non-executable', 'text'}),
+        ('gpg', {'salt', 'file', 'non-executable', 'text', 'gnupg'}),
+        ('jinja', {'salt', 'file', 'non-executable', 'text', 'jinja'}),
+        ('jinja|py', {'salt', 'file', 'non-executable', 'text', 'jinja'}),
+        ('jinja|yaml', {'salt', 'file', 'non-executable', 'text', 'jinja'}),
+        (
+            'jinja|yaml|gpg', {
+                'salt', 'file', 'non-executable', 'text', 'jinja',
+            },
+        ),
```

Comment:
This does have the benefit of files with the extension `.gpg` being marked as binary while files with the `gpg` interpreter still get marked as text.

Reply:
I apologize, I quotes the wrong section. The relevant line is 153:

`('gpg', {'salt', 'file', 'non-executable', 'text', 'gnupg'}),`

Reply:
Nice!